src/main/resources/templates/scala-lang/parametrized_input.ftl

```diff
@@ -54,6 +54,16 @@ case class ${className}(
 </#if>
 ) extends GraphQLParametrizedInput {
 
+    override def deepCopy(): ${className} {
+        val parametrizedInput = ${className}()
```

Comment:
Unable to use new, and then assign values ​​one by one.
should be
A(
field1,field2,....
)